velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1399,4 +1405,42 @@ struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
   }
 };
 
+template <typename T>
+struct ToISO8601FromTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  std::string sessionTzName = "";
+  std::string tzOffsetStr = "";
+  const date::time_zone* sessionTimezone = nullptr;
+  int64_t targetTimezoneID = -1;
```

Comment:
This is not used.

Reply:
Thanks, this was leftover from prior implementation. Removed.